backend/routers/field_officer.py

```
@@ -256,138 +287,163 @@ def officer_check_out(request: OfficerCheckOutRequest, db: Session = Depends(get
+            message=f"Successfully uploaded {len(images)} images",
         )
-        
+
     except HTTPException:
         raise
     except Exception as e:
         logger.error(f"Error uploading visit images: {e}", exc_info=True)
-        raise HTTPException(status_code=500, detail="Image upload failed. Please try again.")
+        raise HTTPException(
+            status_code=500, detail="Image upload failed. Please try again."
+        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)